### PR TITLE
Transform migration version file name to string

### PR DESCRIPTION
**Pull request by Lexinek**
Fixes TypeError with integer type

### CODE REVIEW

File: src/EventSubscriber/ChangeCodingStandardEventSubscriber.php [hunk 1]
```diff
@@ -69,12 +69,12 @@ private function getCurrentMigrationFileName() : string
...
 
 		$i = 0;
-		while ( ! file_exists($this->getMigrationFileByVersion($version)) && $i <= 10) {
+		while ( ! file_exists($this->getMigrationFileByVersion((string) $version)) && $i <= 10) {
 			$version--;
 			$i++;
 		}
 
-		$path = $this->getMigrationFileByVersion($version);
+		$path = $this->getMigrationFileByVersion((string) $version);
```
Comment (enumag) [inline src/EventSubscriber/ChangeCodingStandardEventSubscriber.php]: Maybe getMigrationFileByVersion should accept int instead. What do you think @TomasVotruba ?

Comment (Lexinek) [inline src/EventSubscriber/ChangeCodingStandardEventSubscriber.php]: The first call `$version` is string, but any next is integer due to `$version--`.

**Comment by TomasVotruba** [comment]
Could you send it to monorepo please? https://github.com/Zenify/Zenify

Also failing test would be great.

Not sure what does method should do :D

**Comment by TomasVotruba** [comment]
On the other hand, I wonder if this coding standard check has some meaning.

It only fixes spaces/tabs and coding standards are much more complex.

What to you think about removing this feature completely?

**Comment by Lexinek** [comment]
@TomasVotruba not sure what do you mean, are you talking about the Scrutinizer?
Anyways i will have time later today, i will move the PR to the main repository as proposed, as well i will try to write the test.

**Comment by TomasVotruba** [comment]
@Lexinek I'm talking about Code Sniffer or PHP-CS-Fixer. Tools that force certain coding standard (space after comma, 1 line between methods etd.).

Do you have these in your projects?

**Comment by TomasVotruba** [comment]
What do you think @enumag @TomasPilar @f3l1x about dropping this coding-standard spaces/tabs only feature?

**Comment by enumag** [comment]
@TomasVotruba Personally I'm using spaces everywhere now so I wouldn't mind. But other Nette developers might want it so I'd keep it for BC reasons.

**Comment by Lexinek** [comment]
@TomasVotruba not yet, we are using the PhpStorm features (code check, code rearrange, cleanup, etc.), but i have already seen this on Slack and i might give it a chance! ;)

**Comment by f3l1x** [comment]
@TomasVotruba I think that coding standard should be as strict as possible. :-) I vote for keep it.

**Comment by TomasVotruba** [comment]
@enumag The issue is about mixing of concerns. According to my Coding standard, this migration is missing a lot, even when applying spaces.

@f3l1x This is exactly about making use of strict coding standard. E.g. this file is missing `strict_types` definition. It should be role of coding standard of checking/fixing it, not role of Migrations.

@Lexinek You will love it. It much more powerful than bare PHPStorm. More over it will help you to focus on important stuff in code reviews.

**Comment by enumag** [comment]
Well yes, for strict coding standards you'd have to be able to set your own template for the generated migration. That might be possible with some hacking or a PR to doctrine/migrations, I just never had a reason to do that.

Anyway is there a reason to remove this feature? I agree it's something that doesn't belong to this package I don't see any harm in keeping it.

**Comment by TomasVotruba** [comment]
Exactly my point. Coding standard is not trying to migrate database, does it? :)

The reason is:
https://en.wikipedia.org/wiki/Broken_windows_theory

Also now it would also remove a bug. I would not care about it without this issue.

> I agree it's something that doesn't belong to this package 

This is the best reason.

**Comment by enumag** [comment]
@TomasVotruba How about we ask on Nette forum or Twitter to get feedback from more users? Mainly to see how many people are using it and consider it important.

**Comment by TomasVotruba** [comment]
@enumag Sure. Feel free to do so.

Actually, I think we were the only one who had coding standard issue :D

**Comment by enumag** [comment]
@TomasVotruba That might as well be true. :-D Alright I'll ask on the forum.

**Comment by Lexinek** [comment]
Resolved by Zenify/Zenify#16

**Pull request closed without merging**